charmbox-setup.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -ex
+
+# Bundletester and friends in tims ppa
+sudo add-apt-repository -y ppa:tvansteenburgh/ppa
+
+sudo apt-get update -qqy
+sudo apt-get install -qy  \
+                     build-essential \
+                     juju-deployer \
+                     libssl-dev \
+                     make \
+                     python-dev \
+                     python-jujuclient \
+                     python3-dev \
+                     python-pip \
+                     python3-pip \
+                     python-virtualenv \
```

Comment:
I thought jujubox did this, but when I checked it installs a different package.
```
$ apt-cache search virtualenvwrapper
virtualenvwrapper - extension to virtualenv for managing multiple virtual Python environments
$ apt-cache search python-virtualenv
python-virtualenv - Python virtual environment creator
```
We probably don't need that in jujubox any longer. I will create a bug to track this in jujubox.